src/crypto-providers/ledgerCryptoProvider.ts

```diff
@@ -839,14 +838,14 @@ export const LedgerCryptoProvider: (transport: Transport) => Promise<CryptoProvi
     const response = await ledger.signTransaction(dummyTx)
     if (!response.auxiliaryDataSupplement) throw Error(Errors.MissingAuxiliaryDataSupplement)
 
-    return encodeVotingRegistrationMetaData(
+    return encodeCIP36RegistrationMetaData(
```

Comment:
I see you used `CIP36Registration` in this renaming (whereas ledgerjs uses mostly `cip36VoteRegistration` - there is the `vote`) - fine by me, I just wanted to point this out in case it was not intentional.